ada_2_fhir/zibs2017/payload/zib-plannedcareactivityfortransfer.xsl

```diff
@@ -0,0 +1,432 @@
+        <xsl:param name="in" as="element()?"/>
+        <xsl:param name="logicalId" as="xs:string?"/>
+        <xsl:for-each select="$in">
+            <xsl:variable name="resource">
+                <xsl:variable name="profileValue">http://fhir.nl/fhir/StructureDefinition/nl-core-location</xsl:variable>                
+                <Location>
+                    <xsl:if test="string-length($logicalId) gt 0">
+                        <xsl:choose>
+                            <xsl:when test="$referById">
+                                <id value="{nf:make-fhir-logicalid(tokenize($profileValue, './')[last()], $logicalId)}"/>
+                            </xsl:when>
+                            <xsl:otherwise>
+                                <id value="{$logicalId}"/>
+                            </xsl:otherwise>
+                        </xsl:choose>
+                    </xsl:if>
+                    
+                    <meta>
+                        <profile value="{$profileValue}"/>
+                    </meta>
+                    
+                    <xsl:for-each select="(zorgaanbieder_identificatienummer | zorgaanbieder_identificatie_nummer | healthcare_provider_identification_number | identification_number | zibroot/identificatienummer | hcimroot/identification_number)[@value]">
+                        <identifier>
+                            <xsl:call-template name="id-to-Identifier">
+                                <xsl:with-param name="in" select="."/>
+                            </xsl:call-template>
+                        </identifier>
+                    </xsl:for-each>
+                    
+                    <xsl:variable name="organizationLocation" select="(organisatie_locatie | organization_location)/@value"/>
+                    <xsl:if test="$organizationLocation">
+                        <!-- Cardinality of ADA allows organizationLocation to be present without organizationName. This allows Organization.name to be the value of organizationLocation. This conforms to mapping of HCIM HealthcareProvider -->
+                        <name value="{$organizationLocation}"/>
+                    </xsl:if>
+                    
+                    <!-- J.D. - Noticed in zib2017 ADA project that in vanilla ADA contact and address are in a container with the same name. Filtering here ... -->
+                    <xsl:variable name="contact" as="element()*">
+                        <xsl:choose>
+                            <xsl:when test="(contactgegevens | contact_information)/(contactgegevens | contact_information)">
+                                <xsl:sequence select="(contactgegevens | contact_information)/(contactgegevens | contact_information)"/>
+                            </xsl:when>
+                            <!-- Add zib-Payer contact -->
```

Comment:
Is logic related to zib-Payer really necessary here, as this template is only used within the context of PlannedCareActivityForTransfer? Also see template `match` in line 342.